app/documents/nih_report.rb

```diff
@@ -0,0 +1,244 @@
+      data << ["<b>Threshold's FPR</b>", @options[:threshold_fpr]&.round(3)]
+    end
+
+    # transpose data
+    data = data.transpose
+
+    text "Summary", size: 15, style: :bold, indent_paragraphs: 60
+    move_down 10
+
+    table data, position: :center, width: 650 do
+      cells.style do |cell|
+        cell.border_width = 10
+        cell.border_color = "FFFFFF"
+        cell.padding = 5
+        cell.size = 10
+        cell.inline_format = true 
+        cell.align = :center
+      end
+    end
+    
+    move_down 30
+  end
+
+  def render_confussion_matrix
+    move_down 10
+    data = [
+      ["", "PREDICTED\nNEGATIVE", "PREDICTED\n POSITIVE", "TOTAL"],
+      [
+        "ACTUAL\nNEGATIVE", 
+        "<font size='18'>#{@confusion_matrix[:true_negative]}</font>\nTrue Negative",
+        "<font size='18'>#{@confusion_matrix[:false_positive]}</font>\nFalse Positive",
+        "<font size='18'>#{@confusion_matrix[:true_negative] + @confusion_matrix[:false_positive]}</font>\nActual Negative"
+      ],
+      [
+        "ACTUAL\nPOSITIVE",
+        "<font size='18'>#{@confusion_matrix[:false_negative]}</font>\nFalse Negative",
+        "<font size='18'>#{@confusion_matrix[:true_positive]}</font>\nTrue Positive",
+        "<font size='18'>#{@confusion_matrix[:false_negative] + @confusion_matrix[:true_positive]}</font>\nActual Positive"
+      ],
+      [
+        "TOTAL",
+        "<font size='18'>#{@confusion_matrix[:true_negative] + @confusion_matrix[:false_negative]}</font>\nPredicted Negative",
+        "<font size='18'>#{@confusion_matrix[:false_positive] + @confusion_matrix[:true_positive]}</font>\nPredicted Positive",
+        "<font size='18'>#{@confusion_matrix[:true_negative] + @confusion_matrix[:false_positive] + @confusion_matrix[:false_negative] + @confusion_matrix[:true_positive]}</font>\nTotal"
+      ]
+    ]
+
+    cells = [
+      [
+        RotatedCell.new(@document, [0, 0], content: data[0][0]), 
+        TitleCell.new(@document, [0, 0], content: data[0][1]), 
+        TitleCell.new(@document, [0, 0], content: data[0][2]), 
+        TitleCell.new(@document, [0, 0], content: data[0][3])
+      ],
+      [
+        RotatedCell.new(@document, [0, 0], content: data[1][0]), 
+        GreyCell.new(@document, [0, 0], content: data[1][1]), 
+        GreyCell.new(@document, [0, 0], content: data[1][2]), 
+        TotalCell.new(@document, [0, 0], content: data[1][3])
+      ],
+      [
+        RotatedCell.new(@document, [0, 0], content: data[2][0]), 
+        GreyCell.new(@document, [0, 0], content: data[2][1]), 
+        GreyCell.new(@document, [0, 0], content: data[2][2]), 
+        TotalCell.new(@document, [0, 0], content: data[2][3])
+      ],
+      [
+        RotatedCell.new(@document, [0, 0], content: data[3][0]), 
+        TotalCell.new(@document, [0, 0], content: data[3][1]), 
+        TotalCell.new(@document, [0, 0], content: data[3][2]), 
+        TotalCell.new(@document, [0, 0], content: data[3][3])
+      ]
+    ]
```

Comment:
Oh, I really like those specific cell classes! They really help to simplify.

Hum, maybe we can go even further, and merge `data` into `cells`, and move the `<font>` declaration into `GreyCell` and `TotalCell` that would take distinct kwargs?

```ruby
TotalCell.new(@document, title: "XYZ", value: 12)
```

Note: `point` is also always the same `[0, 0]`, maybe it could be right into `CustomCell`?